pcp/versions/2.0/inventory.md

```diff
@@ -11,32 +11,69 @@ A client node that wants an inventory list of a PCP broker must 1) be
 connected to it (refer to [sessions][1]) and 2) send an inventory
 request message to a PCP broker.
 
-In case the request is processed successfully, the broker will reply with an
-inventory response message.
+If the request is processed successfully, the broker will reply with an
+inventory response message. Additionally, if the inventory request includes a
+`subscribe_to_updates` flag set to `true`, the broker will send the client an
+`inventory update` message when the broker's inventory changes. This implies
```

Comment:
The note about two registries isn't needed here, just musing on my part. We keep implementation details out of the protocol description.